gadgets/trace_exec/gadget.yaml

```diff
@@ -31,20 +31,20 @@ datasources:
         annotations:
           description: Thread ID
           template: pid
-      uid:
+      user_raw:
```

Comment:
Maybe we should still use uid_raw/gid_raw/loginuid_raw?

Reply:
I changed the name to have the column name be `user` instead of `uid` when we are actually displaying usernames. Would be probably best if the enriched name is `user` and the raw name is `uid(_raw)` as you mentioned.

Is there a way to easily achieve that?

Reply:
Hmm, makes sense - but then we also could use `uid` without the `_raw` suffix. So maybe:

* look for a target annotation
* if none is given, replace string `uid` with `user` (`gid` with `group`) and try using that for the target field name - so that it also works for `loginuid` to `loginuser`